src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -310,6 +310,10 @@
                 <source>This value does not match the expected {{ charset }} charset.</source>
                 <target>Detta värde har inte den förväntade teckenkodningen {{ charset }}.</target>
             </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>Detta är inte en giltlig BIC-kod.</target>
```

Comment:
Google Translate says that `giltlig` should be `giltig`  Is this a false positive?